src/Management/src/Endpoint/Actuators/Loggers/LoggersEndpointMiddleware.cs

```diff
@@ -92,17 +89,20 @@ private async Task<Dictionary<string, string>> DeserializeRequestAsync(Stream st
         return [];
     }
 
-    protected override async Task WriteResponseAsync(LoggersResponse result, HttpContext context, CancellationToken cancellationToken)
+    protected override async Task WriteResponseAsync(LoggersResponse? result, HttpContext context, CancellationToken cancellationToken)
     {
-        if (result.HasError)
+        if (result is { HasError: true })
         {
             context.Response.StatusCode = (int)HttpStatusCode.BadRequest;
         }
         else
         {
-            context.HandleContentNegotiation(_logger);
-            JsonSerializerOptions options = ManagementOptionsMonitor.CurrentValue.SerializerOptions;
-            await JsonSerializer.SerializeAsync(context.Response.Body, result.Data, options, cancellationToken);
+            if (result != null)
```

Comment:
Can we return `204 No Content` when there is success and no response body? That would be more correct and match how Spring works